### ISSUE TITLE
Testing bug - Content page patterns

### ISSUE THREAD

**Issue by sarahlazarich** [bloom-housing/bloom#4809]
## Background
Current Behavior 
- Navigate to the accessibility statement page from the footer on the https://bloom-detroit.netlify.app/accessibility
- Navigate to the feedback page from the footer https://bloom-detroit.netlify.app/feedback
- Notice that the page is themed for core (blue page header, wrong font) 

Expected Behavior 
- The accessibility statement should be themed for Detroit
- Example: https://bloom-detroit.netlify.app/disclaimer

## Technical Notes
- We need to pull the new PageHeader pattern from `PageHeaderLayout`

## Acceptance Criteria
- [Accessibility statement page](https://bloom-detroit.netlify.app/accessibility) uses the Detroit colors
- [Accessibility statement page](https://bloom-detroit.netlify.app/accessibility) uses the correct font
- Feedback page should have the correct colors and fonts

**Comment by jaredcwhite** [comment]
Making a note here that this is a page unique to Detroit, it's not in Core.

**Comment by sarahlazarich** [comment]
@jaredcwhite where should this be QA'd?

**Comment by sarahlazarich** [comment]
@jaredcwhite the first link https://bloom-detroit.netlify.app/accessibility doesn't look themed

**Comment by sarahlazarich** [comment]
this looks good!
